server/channels/app/export.go

```diff
@@ -1130,14 +1142,18 @@ func (a *App) exportAllDirectPosts(ctx request.CTX, job *model.Job, writer io.Wr
 	return attachments, nil
 }
 
-func (a *App) exportFile(outPath, filePath string, zipWr *zip.Writer) *model.AppError {
+func (a *App) exportFile(outPath, filePath string, zipWr *zip.Writer, rctx request.CTX ) *model.AppError {
```

Comment:
` request.CTX` should always be the first parameter. Can you please update the code?